scripts/ci_install_dependency.sh

```diff
@@ -2,6 +2,9 @@
 Install the dependency in CI.
 """
 
+# Use repo from environment variable, passed from GitHub Actions
+FLASHINFER_REPO="${FLASHINFER_REPO:-https://flashinfer.ai/whl/cu121/torch2.4/flashinfer/}"
+
```

Comment:
I force reinstall flashinfer in the file below

Reply:
ok